bliss/models/decoder.py

```diff
@@ -509,13 +491,16 @@ def _adjust_psf(self):
 
 
 class GalaxyTileDecoder(nn.Module):
-    def __init__(
-        self, tile_slen, ptile_slen, n_bands, galaxy_decoder: galaxy_net.CenteredGalaxyDecoder
-    ):
+    def __init__(self, tile_slen, ptile_slen, n_bands, galaxy_model: GalaxyModel):
         super().__init__()
         self.n_bands = n_bands
         self.tiler = Tiler(tile_slen, ptile_slen)
         self.ptile_slen = ptile_slen
+
+        if isinstance(galaxy_model, OneCenteredGalaxyAE):
+            galaxy_decoder = galaxy_model.get_decoder()
+        elif isinstance(galaxy_model, GalsimGalaxyDecoder):
+            galaxy_decoder = galaxy_model
```

Comment:
I'll add that - I'm surprised mypy didn't catch the unbound error

Reply:
Based on the types, MyPy says this will never happen:
![image](https://user-images.githubusercontent.com/12701492/166821947-878459db-173c-4af6-83ed-052acc6bd049.png)

Reply:
Added